modules/acquia_cms_tour/acquia_cms_tour.routing.yml

```diff
@@ -1,10 +1,3 @@
-acquia_cms_tour.tour:
-  path: '/admin/tour'
-  defaults:
-    _controller: '\Drupal\acquia_cms_tour\Controller\TourController::build'
-  requirements:
-    _permission: 'access acquia cms tour'
-
 acquia_cms_tour.enabled_modules:
```

Comment:
@apathak18 - Lets use route `acquia_cms_tour.tour` instead of `acquia_cms_tour.enabled_modules`.
cc: @vishalkhode1 @vipin-mittal-acquia

Reply:
Or more precisely we can create `acquia_cms_tour.wizard`